### DISCUSSION TITLE
Feature request: Functionality to clear link click count

### DISCUSSION THREAD

**Discussion by Ilosariph** [JulianPrieber/littlelink-custom#446] [Feature requests]
When installing LinkStack, I clicked on a few links myself. A button to reset the click count of a link (or of all links) would be great.

**Comment by JulianPrieber** [comment]
This is pretty low on our list of features to implement, but might come sooner or later.
Currently, the only thing you can do is either delete the link and create a new one or export your links on the settings page, edit the downloaded file and change all clicks to 0, save and upload the file again and your clicks have been reset.

**Comment by Tiritibambix** [comment]
Hello! I am writing to follow up on the feature requested on May 21, 2023. I am eager to know if there have been any updates or if there is an estimated timeline for its implementation. The current workarounds are quite cumbersome and time-consuming, so any progress on this would be greatly appreciated. Thank you!
